docs/source/samples/15_rgb_mobilenet_4k.rst

```diff
@@ -0,0 +1,34 @@
+15 - 4K RGB MobileNetSSD
+========================
+
+Demo
+####
+
+.. raw:: html
+
+    <div style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
+        <iframe src="//www.youtube.com/embed/Ww-vLSxOdW8" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
```

Comment:
missing `https://`

Reply:
Yah, I noticed that, but figured it was added somewhere since all of the examples are missing it. Should I be merging into `gen2_develop`?